ogr/ogrsf_frmts/dwg/ogr_dwg.h

```diff
@@ -124,6 +124,7 @@ class OGRDWGLayer final: public OGRLayer
     OGRFeature *        TranslateHATCH( OdDbEntityPtr poEntity );
     OGRFeature *        TranslateTEXT( OdDbEntityPtr poEntity );
     OGRFeature *        TranslateINSERT( OdDbEntityPtr poEntity );
+	OGRFeature *        Translate3DFACE(OdDbEntityPtr poEntity);
```

Comment:
using 4-space indentation instead of tabulation:
```suggestion
     OGRFeature *        Translate3DFACE(OdDbEntityPtr poEntity);
```

Reply:
sorry first commit, i did not knew the code format you use, i apply changes on tabs